src/domain/pagination/types.ts

```diff
@@ -1,21 +1,15 @@
 import {
   ApolloQueryResult,
-  DocumentNode,
   FetchMoreOptions,
   FetchMoreQueryOptions,
-  TypedDocumentNode,
+  OperationVariables,
 } from "@apollo/client";
 
 // Copied from Apollo's type definitions
```

Comment:
Is this copied from Apollo?

Reply:
not 1:1 copy but still mostly copied from there